public/docs/dart/latest/guide/forms.jade

```diff
@@ -0,0 +1,662 @@
+:marked
+  ## Create an initial HTML form template
+
+  Create a new file under `lib` called `hero_form_component.html`,
+  and put the following template code in it:
+
++makeExample('forms/dart/angular2_forms-2-COMPONENT/lib/hero_form_component.html', null, 'lib/hero_form_component.html')
+
+:marked
+  That is plain old HTML 5. We're presenting two of the `Hero` fields, `name` and `alterEgo`, and
+  opening them up for user input in input boxes.
+
+  The Name `<input>` control has the HTML5 `required` attribute;
+  the Alter Ego `<input>` control does not because `alterEgo` is optional.
+
+  We've got a Submit button at the bottom with some classes on it.
+
+  **We are not using Angular yet**. There are no bindings. No extra directives. Just layout.
+
+
+  The `container`,`form-group`, `form-control`, and `btn` classes are [Bootstrap](http://getbootstrap.com/) CSS. Purely cosmetic.
+  We're using Bootstrap to gussy up our form.
+  Hey, what's a form without a little style!
+
+.callout.is-important
+  header Angular forms do not require a style library
+  :marked
+    Angular makes no use of the `container`, `form-group`, `form-control`, and `btn` classes or
+    the styles of any external library. Angular apps can use any CSS library
+    ... or none at all.
+
+:marked
+  Let's add the stylesheet.
+
+  1. Download the Bootstrap stylesheet from
+  https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css,
+  and put it in the `web` directory.
+
+  2. Edit `web/index.html`, adding a link to `bootstrap.min.css`:
+
+- var stylePattern = { otl: /(&lt;link rel.*$)/gm };
++makeExample('forms/dart/angular2_forms-2-COMPONENT/web/index.html', 'css', 'web/index.html', stylePattern)(format="linenums:4")
+
+  [PENDING: runnable now? Remind about pub get? remind them to look in the browser console]
+
+
+.l-main-section
+:marked
+  ## Add powers with ***ng-for**
+  Our hero must choose one super power from a fixed list of Agency-approved powers.
+  We maintain that list internally (in `HeroFormComponent`).
+
+  We'll add a `select` to our
+  form and bind the options to the `powers` list using `NgFor`,
+  a technique used before in [Displaying Data](./displaying-data.html).
+
+  Add the following HTML *immediately below* the Alter Ego group.
++makeExample('forms/dart/angular2_forms-3-BINDING/lib/hero_form_component.html', 'powers', 'lib/hero_form_component.html')(format="linenums:14")
+
+:marked
+  This code repeats the `<option>` tag for each power in the list of powers.
+  The `#p` local template variable is a different power in each iteration;
+  we display its name using the interpolation syntax with the double curly braces.
+
+
+.l-main-section
+:marked
+  ## Two-way data binding with ***ng-model**
+  Running the app right now would be disappointing.
+
+figure.image-display
+  img(src="/resources/images/devguide/forms/hf-3.png" alt="Early form with no binding")
+:marked
+  The problem is that we aren't binding to the `Hero` yet.
```

Comment:
You did well to replace my sentence. But you didn't state the problem.

I changed this sentence in the TS to: "We don't see hero data because we are not binding to the `Hero` yet."